vars/withGithubCheck.groovy

```diff
@@ -66,6 +66,50 @@ def call(Map args = [:], Closure body) {
   }
 }
 
+String detailsUrl(redirect, isBo=false) {
```

Comment:
This should be provided as isolated step then others can consume this, such as the withGithubStatus.